src/dashboard/form-manager/components/FieldRenderer.tsx

```
@@ -0,0 +1,152 @@
+                        } }
+                    />
+                ) }
+            </MediaUploader>
+        </CustomField>
+    );
+};
+
+export const getFieldConfig = ( field: FormField, section: Section ) => {
+    const mappedField = {
+        ...field,
+        label: <span dangerouslySetInnerHTML={ { __html: field.title } } />,
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)